### DISCUSSION TITLE
How to define local coordinate systems for the knee joint and calculate knee joint angles from a kinematic acquisition

### DISCUSSION THREAD

**Discussion by Yokonie** [kineticstoolkit/kineticstoolkit#130] [1. Users Q&A]
Hello Félix Chénier,

I was trying to get hip, knee and ankle joint angles from a kinematic acquisition and was able to calculate hip and ankle joint angles based upon the instruction given and papers that you referenced (ISB recommendation on definitions of joint coordinate systems of various joints). However, these papers by Wo et al. do not provide any recommendation on how to define local coordinate systems for the knee joint and calculate knee joint angles. 

I was wondering if you have any suggestion on how I should define local coordinate systems for the knee joint or know any articles that provide the recommendation that I can use.
Thank you!

Yoshi

**Comment by felixchenier** [comment]
@Yokonie 

Hello Yoshi

I don't really work on the lower body so I couldn't tell much. I know that a lot of people still refer to this paper that is getting quite old now:

Grood, E.S., Suntay, W.J., 1983. A Joint Coordinate System for the Clinical Description of Three-Dimensional Motions: Application to the Knee. Journal of Biomechanical Engineering 105, 136–144. https://doi.org/10.1115/1.3138397

but unfortunately I couldn't personally guide you more.

Félix

**Reply by Yokonie** [reply]
Hello Félix Chénier,

Thank you for the reply and putting an article here. I will have a look at it and try to figure out how to calculate knee joint angles.

Thank you very muchj!

Yoshi
